whisper/stt/processing/streaming.py

```diff
@@ -62,19 +68,28 @@ async def wssDecode(ws: WebSocketServerProtocol, model_and_alignementmodel):
             logger.info(f"End of stream {message}")
             await ws.close(reason="End of stream")
             break
-        online.insert_audio_chunk(bytes_to_array(message))
-        o, _ = online.process_iter()
-        logger.info(o)
-        await ws.send(whisper_to_json(o))
+        audio_chunk = bytes_to_array(message)
+        online.insert_audio_chunk(audio_chunk)
+        logger.info(f"{online.get_buffer_size()}>{STREAMING_MIN_CHUNK_SIZE}= {online.get_buffer_size() > STREAMING_MIN_CHUNK_SIZE}")
+        if online.get_buffer_size() > STREAMING_MIN_CHUNK_SIZE:
+            o, p = online.process_iter()
+            logger.info(o)
+            await ws.send(whisper_to_json(p, partial=True))
+            await ws.send(whisper_to_json(o))
+        else:
+            logger.info(f"Chunk too small {len(audio_chunk)/sample_rate}<{STREAMING_MIN_CHUNK_SIZE} (added {len(audio_chunk)/sample_rate}), skipping")
+            await ws.send(whisper_to_json((None, None, "")))
 
 
 def ws_streaming(websocket_server: WSServer, model_and_alignementmodel):
     """Sync Decode function endpoint"""
     res = websocket_server.receive(timeout=10)
     try:
         config = json.loads(res)["config"]
-        sample_rate = config["sample_rate"]
         logger.info(f"Received config: {config}")
+        sample_rate = config["sample_rate"]
+        if sample_rate != 16000:
+            raise NotImplementedError("Only 16000 sample rate is supported")
```

Comment:
Add a comment to mention that what is not implemented is the conversion before sending to whisper network
(but VAD and all the logic around timeframe management already work with any sample rate)